charts/snapshot-controller/values.yaml

```diff
@@ -41,3 +41,11 @@ serviceAccount:
   # Specifies whether a ServiceAccount should be created
   create: true
   name: ""
+
+volumeSnapshotClasses: []
+#  - name: ebs-csi-aws-delete
+#    annotations:
+#      snapshot.storage.kubernetes.io/is-default-class: "true"
+#    driver: ebs.csi.aws.com
+#    deletionPolicy: Delete
```

Comment:
I don't really see this as responsibility of the snapshot-controller. Without some CSI driver you don't need any snapshot classes. Setting up snapshot classes should be the responsibility of that CSI driver deployment, not the generic snapshot-controller.

Reply:
The external-shapshotter is the source of record for the CRD definition. I set the default of the chart not to create snapshot classes. I have included the commented out lines as an example. In general CSI drivers can function without a snapshot controller so they may not want to include a feature for creating resources for a external/optional dependency. I included it for convenience so a separate chart would not be needed. If you feel strongly it should not be here, I can remove it.